app/services/iiif_manifest.rb

```diff
@@ -2,7 +2,7 @@ class IiifManifest < ::Spotlight::Resources::IiifManifest
   def to_solr
     add_noid
     # this is called in super, but idempotent so safe to call here also; we need the metadata
```

Comment:
does this comment reference the line you deleted?

Reply:
Nah, it's accurate for the existing line too.